testsuite/features/support/http_client.rb

```diff
@@ -0,0 +1,78 @@
+# Copyright (c) 2022 SUSE LLC.
+# Licensed under the terms of the MIT license.
+
+require 'faraday'
+
+# Wrapper class for HTTP client library (Faraday)
+class HttpClient
+  def initialize(host)
+    puts 'Activating HTTP API'
+    @http_client = Faraday.new('https://' + host)
+  end
+
+  # rubocop:disable Metrics/AbcSize
+  # rubocop:disable Metrics/CyclomaticComplexity
+  # rubocop:disable Metrics/PerceivedComplexity
+  # rubocop:disable Metrics/MethodLength
```

Comment:
Go ahead and add the whole file as exceptions in .rubocop_todo.yml :)

Reply:
ok, i did not want to hide the dust under the carpet :-)